validator/validate.py

```diff
@@ -66,9 +66,9 @@ def validate(path, format='json',
         # path (string).
         with open(approved_applications) as approved_apps:
             apps = json.load(approved_apps)
-        # Add the '*' fake max version. Used specifically for WebExtensions,
-        # see https://github.com/mozilla/amo-validator/issues/390.
+        # Add the '*' fake max version for Desktop and Android.
         apps['1']['versions'].append('*')
+        apps['7']['versions'].append('*')
```

Comment:
this code shouldn't be copy/pasted. Maybe it's only needed in `validate()` and not needed in `validateor/constants.py` ? If it's really truly needed in both places you could make a shared utility function to add the `*`.